packages/opentelemetry-core/test/common/time.test.ts

```diff
@@ -111,7 +111,11 @@ describe('time', () => {
     it('should convert Date hrTime', () => {
       const timeInput = new Date();
       const output = timeInputToHrTime(timeInput);
-      assert.deepStrictEqual(output, [timeInput.getTime(), 0]);
+      const epochSeconds = timeInput.getTime() / 1000;
+      const seconds = Math.trunc(epochSeconds);
```

Comment:
This is basically the code of `numberToHrtime` function. In unit test I would rather expect something like this
```
const timeInput = new Date(1609256394099);
const output = timeInputToHrTime(timeInput);
assert.deepStrictEqual(output, [1609......., 099........])
```

Reply:
Nit: you can even create a loop with few different date values and expected output - this is just optional

Reply:
unit test has been changed, check it again plz